src/lib/input/input-container.spec.ts

```diff
@@ -1020,6 +1136,29 @@ class MdInputContainerWithFormErrorMessages {
 
 @Component({
   template: `
+    <form #form="ngForm" novalidate>
+      <md-input-container>
+        <input mdInput
+            [formControl]="formControl"
```

Comment:
`NgForm` comes with the `FormsModule` and `formControl` comes from the `ReactiveFormsModule`. It's not really great to mix them, and the point of using `formControl` is to have a standalone control *without* a parent. 

If you want a more "idiomatic" test case, I'd suggest switching to `formGroup` on the form tag and then use `formControlName` rather than `formControl`.